src/components/feedback/FeedbackWidget.vue

```diff
@@ -273,6 +287,17 @@
         this.feedbackInputState = false;
       },
 
+      wordCount() {
+        if (!this.feedback) {
```

Comment:
We should be able to remove this and rely solely on the 5 word validation below.